packages/nest/src/migrations/update-13-2-0/update-to-nest-8.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  formatFiles,
+  installPackagesTask,
+  readJson,
+  Tree,
+  updateJson,
+} from '@nrwl/devkit';
+import { sortObjectByKeys } from '@nrwl/tao/src/utils/object-sort';
+import { checkAndCleanWithSemver } from '@nrwl/workspace';
+import { satisfies } from 'semver';
+import {
+  nestJsSchematicsVersion,
+  nestJsVersion8,
+  rxjsVersion7,
+} from '../../utils/versions';
+
+export default async function update(tree: Tree) {
+  const shouldUpdate = await isUpdatable(tree);
+
+  if (!shouldUpdate) {
+    return;
+  }
+
+  updateVersion(tree);
+
+  await formatFiles(tree);
+
+  return (): void => {
+    installPackagesTask(tree);
```

Comment:
issue: @gperdomor apparently this doesn't work, consumers still need to run install package manually. I think we can log something here instead. Check out the `storybook` migrations

Reply:
Pushed the change

Reply:
I see you already fixed that 👍🏻